packages/client/src/views/Settings/SettingsPage.tsx

```diff
@@ -188,6 +195,18 @@ class SettingsView extends React.Component<IProps, IState> {
     }
   }
 
+  componentDidMount() {
+    let phonedNumberUpdated = false
+    const historyState = this.props.history.location.state as any
```

Comment:
Instead of casting this to any, you can also pass the type variables for `RouteComponentProps` so you get a right type of structure here. So something like:

```ts
type IProps = IntlShapeProps &
  RouteComponentProps<{}, {}, { phonedNumberUpdated: boolean }> & {
```

Reply:
I have updated the code and remove any as type.